crates/revm/src/instructions/arithmetic.rs

```diff
@@ -1,45 +1,59 @@
 use super::i256::{i256_div, i256_mod};
-use crate::{gas, Interpreter, Return, Spec, U256};
+use crate::{gas, Host, Interpreter, Return, Spec, U256};
 
-pub fn div(op1: U256, op2: U256) -> U256 {
-    op1.checked_div(op2).unwrap_or_default()
+pub fn wrapped_add(interpreter: &mut Interpreter, _host: &mut dyn Host) {
```

Comment:
Are we Ok with dyn dispatch in all opcodes? Looks like we have it inside the it the interp loop and it may be impactful for performance?

Reply:
That is the point, the compiler seems to handle it better when all of call inside `match` has the same sig.

Reply:
Very interesting